hail/python/hail/methods/statgen.py

```diff
@@ -1654,38 +1655,39 @@ def pc_relate(call_expr, min_individual_maf, *, k=None, scores_expr=None,
      - :math:`\widehat{\mu_{is}} \in [0, 1]` be the individual-specific allele
        frequency for individual :math:`i` at genetic locus :math:`s`
 
-     - :math:`{\widehat{\sigma^2_{is}}} := \widehat{\mu_{is}} (1 - \widehat{\mu_{is}})`,
+     - :math:`{\widehat{\sigma^2_{is}}} \coloneqq \widehat{\mu_{is}} (1 - \widehat{\mu_{is}})`,
```

Comment:
Above this, on line 1610 is another `:=`

Reply:
Ah, thanks!

Reply:
Extra thanks, there was a worse typo on that line too!

Reply:
oof, good eye, sum notation included the variance.